test/evergreen.yml

```diff
@@ -5331,12 +5330,13 @@ buildvariants:
 
 # FIXME-WT-13256
 # This variant runs all possible tasks under TSan but instructs TSan to not report any issues it
-# encounters. This allows us to check whether a TSan change may cause other unexpected failures
+# encounters. This allows us to check whether a TSan change may cause unexpected failures (for example
+# if adding a new field to a struct wrapped in #ifdef TSAN changes the struct layout and causes a segfault)
 # while also not having to fix every TSan issue discovered. At the conclusion of the TSan work this
 # variant will be deleted.
 - name: ubuntu2004-tsan-no-throw
   display_name: "~ Ubuntu 20.04 TSAN (No throw)"
-  patch_only: true
+  batchtime: 1440 # 24 hours
```

Comment:
I think once per day is reasonable here, if not less frequently. This is only going to catch edge cases such as adding new struct field which is wrapped in `#ifdef TSAN`, and that change to memory layout reveals a previously hidden bug.

Reply:
Maybe even bump this up to every 3 days? It's effectively mirroring our regular ubuntu builder.

Reply:
From discussion offline we've decided to keep the No Throw variant patch only. This was changed back in [5758e20](https://github.com/wiredtiger/wiredtiger/pull/11548/commits/5758e204d028b66b21286a2c5b2a243d41755d74)